client/src/pages/search/SearchComparision.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { setConfig } from "../../slice/configLoadSlice";
+import { Sidebar } from "./../../layouts/Sidebar";
+import { Layout } from "./../../layouts/Layout";
+import { ModelSelect } from "../../components/modelSelect";
+import { FileSelectComponent } from "../../components/fileSelect";
+// import { socket } from "../../socket";
+
+import { io } from "socket.io-client";
+
+export function ComparisionPage() {
+  // Declare a new state variable, which we'll call "count"
+  const dispatch = useDispatch();
+
+  let serverUrl = "localhost:3000";
+  let socket = io(serverUrl);
+
+  const [isConnected, setIsConnected] = useState(socket.connected);
+  const [benchmark, setBenchmark] = useState({});
+  const [benchmark2, setBenchmark2] = useState({});
+
+  React.useEffect(() => {
+    dispatch(setConfig());
+    socket.connect();
+    const bencmarkListiner = (message) => {
+      console.log("coming here benchmark");
+      setBenchmark(message);
+    };
+    const bencmarkListiner2 = (message) => {
+      console.log("coming here benchmark");
+      setBenchmark2(message);
+    };
+
+    function onConnect() {
+      console.log("connected");
+      setIsConnected(true);
+    }
+    socket.on("connect", onConnect);
+    socket.on("benchmark", bencmarkListiner);
+    socket.on("benchmark2", bencmarkListiner2);
+
+    return () => {
+      socket.off("connect", onConnect);
+      socket.off("benchmark", bencmarkListiner);
+      socket.off("benchmark2", bencmarkListiner2);
+    };
+  }, []);
+
+  const handleClick = (e) => {
+    console.log("click");
+
+    socket.timeout(2000).emit("benchmark", { file: "Beyoncé" }, () => {
```

Comment:
Not too versed with JS, but just wanted to clarify whether "Beyonce" was a default parameter?